src/api/api-client/isi-backend/docs/PagedModelEntityModelVersorgungsquoteSobonHort.md

```
@@ -0,0 +1,38 @@
+console.log(example)
+
+// Convert the instance to a JSON string
+const exampleJSON: string = JSON.stringify(example)
+console.log(exampleJSON)
+
+// Parse the JSON string back to an object
+const exampleParsed = JSON.parse(exampleJSON) as PagedModelEntityModelVersorgungsquoteSobonHort
+console.log(exampleParsed)
+```
+
+[[Back to top]](#) [[Back to API list]](../README.md#api-endpoints) [[Back to Model list]](../README.md#models) [[Back to README]](../README.md)
```

markdownlint-cli2 (0.21.0)

[warning] No empty links

(MD042, no-empty-links)